koku/masu/processor/tasks.py

```diff
@@ -69,8 +69,9 @@
 REFRESH_MATERIALIZED_VIEWS_QUEUE = "reporting"
 REMOVE_EXPIRED_DATA_QUEUE = "remove_expired"
 SUMMARIZE_REPORTS_QUEUE = "process"
-UPDATE_COST_MODEL_COSTS_QUEUE = "reporting"
+UPDATE_COST_MODEL_COSTS_QUEUE = "priority_update"
```

Comment:
We want normal processing to still go to the regular queue, it's just when we call it from the UI. So I think we want to switch https://github.com/project-koku/koku/blob/master/koku/cost_models/cost_model_manager.py#L101 to use the priority queue.